freeglut/freeglut/src/mswin/fg_main_mswin.c

```diff
@@ -686,7 +766,8 @@ static LRESULT fghWindowProcKeyPress(SFG_Window *window, UINT uMsg, GLboolean ke
             if( ToAscii( (UINT)wParam, 0, state, code, 0 ) == 1 )
                 wParam=code[ 0 ];
 
-            INVOKE_WCB( *window, KeyboardUp,
+            if (wParam < 128)
```

Comment:
Will this break existing GLUT functionality? I don't think I've tried Unicode keys so I don't know how this shows up currently.

Reply:
It might break existing programs, but only if they were taking the signed character and casting it to an unsigned value and using it as latin-1. I'm fine with removing this check and maintaining the old 'bad' behavior if you feel it is needed for API stability.

Reply:
Hmm... I might wait to see if someone else has any opinions.

Reply:
Yes I'm afraid this has the potential to break existing programs, and should be avoided.